metron-platform/metron-pcap-backend/src/test/java/org/apache/metron/pcap/integration/PcapTopologyIntegrationTest.java

```diff
@@ -90,23 +90,6 @@ public boolean accept(File dir, String name) {
   }
 
```

Comment:
Test is added back in, but with an `@Ignore` and a comment.  Let me know if you want anything else there.